src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("The truck started working!");

```

Comment:
The message in the `doWork` method should indicate the type of machine that is working. Consider changing the message to something more informative, for example: 'Truck is doing its work.'